components/tableIndexer/pdf_to_images_converter.py

```diff
@@ -0,0 +1,76 @@
+import os
+import time
+import argparse
+from pathlib import Path
+from pdf2image import convert_from_path
+from utils.logger import setup_logging
+import logging
+import sys
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Convert PDF to Images with logging level control."
+    )
+    parser.add_argument(
+        "--log-level",
+        dest="log_level",
+        default="INFO",
+        help="Set the logging level (DEBUG, INFO, WARNING, ERROR, CRITICAL)",
+    )
+    parser.add_argument(
+        "--pdf-name", dest="pdf_name", required=True, help="Specify the PDF file name"
+    )
+    parser.add_argument(
+        "--output-directory",
+        dest="output_directory",
+        default="./outputs/images",
+        help="Specify the output directory for images",
+    )
+    return parser.parse_args()
+
+
+def convert_pdf_to_images(pdf_name, output_directory="./outputs/images"):
```

Comment:
**Nit: Make "./outputs/images" a constant defined once **

I believe you use it a couple times so might as well consolidate